### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by josh671**
added comments

### CODE REVIEW

**Comment by Pei-Hsiau** [comment]
'The greatest mistake is to imagine that we never err.'

File: adder-teams.php [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
-
-<?php   //adder
- 
- //code for quotes starts here
-    //please put your part in the array below. 
-        //Make sure to start the very first word with lower case and add <br> after quote and fill out the form
- $quotes[] = 'from the errors of others, a wise man corrects his own. <br> Please fill out all the fields.';
- $quotes[] = 'errors are not in the art but in the artificers. <br> Please fill out all the fields.';
- $quotes[] = 'There is a disturbance in the Force. <br> Please fill out all the fields. ';
- $quotes[] = 'quotes3';
```
Comment (Pei-Hsiau) [inline adder-teams.php:53]: 'The greatest mistake is to imagine that we never err.'

Comment (Pei-Hsiau) [inline adder-teams.php:53]: 'The greatest mistake is to imagine that we never .'

**Review by ghost — COMMENTED** [review]
'The greatest mistake is to imagine that we never .'

**Review by ghost — COMMENTED** [review]
'The greatest mistake is to imagine that we never err.'

**Pull request merged**